packages/@o3r/components/builders/metadata-check/schema.json

```diff
@@ -0,0 +1,46 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "$id": "ConfigMigrationMetadataCheckBuilderSchema",
+  "title": "Check config migration metadata builder",
+  "description": "",
+  "properties": {
+    "migrationMetadataFolder": {
+      "type": "string",
+      "description": "Path to the file containing the migration metadata."
+    },
+    "granularity": {
+      "type": "string",
+      "description": "Granularity of the migration check.",
+      "default": "minor"
+    },
+    "allowBreakingChanges": {
+      "type": "boolean",
+      "description": "Are breaking changes allowed.",
+      "default": true
+    },
+    "packageManager": {
+      "type": "string",
+      "description": "Override of the package manager, otherwise it will be determined from the project."
```

Comment:
From the angular.json or the process env ? Since you are calling the getPackageManager() function